### DISCUSSION TITLE
这些客户端随机化，又不参与下载/上传的对等方，是网络爬虫吗？

### DISCUSSION THREAD

**Discussion by gentrydeng** [PBH-BTN/PeerBanHelper#768] [问答] [answered]
![image](https://github.com/user-attachments/assets/2c10fd74-0fb6-4377-ad58-dfd4573b0aea)

**Comment by Ghost-chu** [comment, marked as answer]
是的，它们“通常”是DHT/PEX爬虫，通常被Magnet Search Engine等使用，罕见情况下版权方也可能使用类似的技术检测其盗版资源。

对于恶意 Peer 来说，它们通常伪装成类似正常的客户端来让用户放松警惕，而非使用未知或者是随机的PeerID.

![Image_122838855688296](https://github.com/user-attachments/assets/15fe4a9d-109e-4326-a467-36ba20359470)
